staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -39,7 +39,11 @@ var (
 	groupVersions = []schema.GroupVersion{authorization.SchemeGroupVersion}
 )
 
-const retryBackoff = 500 * time.Millisecond
+const (
+	retryBackoff = 500 * time.Millisecond
+	// The maximum length of requester-controlled attributes to allow caching.
+	maxControlledAttrCacheSize = 10000
```

Comment:
How is this bound determined ?
Just curious.

Reply:
It was picked arbitrarily. It's way outside the typical request, but small enough that it would make it harder to DoS the API server with large requests.